WinUIGallery/DataModel/ControlInfoDataSource.cs

```diff
@@ -45,26 +50,26 @@ public ControlInfoDataItem(string uniqueId, string title, string apiNamespace, s
             this.IsNew = isNew;
             this.IsUpdated = isUpdated;
             this.IsPreview = isPreview;
-            this.Docs = new ObservableCollection<ControlInfoDocLink>();
-            this.RelatedControls = new ObservableCollection<string>();
+            this.Docs = docs;
+            this.RelatedControls = relatedControls;
             this.HideSourceCodeAndRelatedControls = hideSourceCodeAndRelatedControls;
         }
 
-        public string UniqueId { get; private set; }
-        public string Title { get; private set; }
-        public string ApiNamespace { get; private set; }
-        public string Subtitle { get; private set; }
-        public string Description { get; private set; }
-        public string ImagePath { get; private set; }
-        public string ImageIconPath { get; private set; }
-        public string BadgeString { get; private set; }
-        public string Content { get; private set; }
-        public bool IsNew { get; private set; }
-        public bool IsUpdated { get; private set; }
-        public bool IsPreview { get; private set; }
-        public bool HideSourceCodeAndRelatedControls { get; private set; }
-        public ObservableCollection<ControlInfoDocLink> Docs { get; private set; }
-        public ObservableCollection<string> RelatedControls { get; private set; }
+        public string UniqueId { get; set; }
```

Comment:
Is the setter public so the JSONSerializer can set them?

Reply:
JsonSerializer will be able to set its value during deserialization. In `private` mode, the number of items in group is 0 (group.Items = 0)

Reply:
I assume protected or internal doesn't work either right?

Reply:
yes not working